src/test/java/com/ot/restaurant/repositories/PersonRepositoryImplTest.java

```diff
@@ -0,0 +1,85 @@
+package com.ot.restaurant.repositories;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.ot.restaurant.builders.DummyPersonBuilder;
+import com.ot.restaurant.entities.Person;
+import com.ot.restaurant.exceptions.IdNullException;
+import com.ot.restaurant.fixtures.PersonFixture;
+import constants.Status;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PersonRepositoryImplTest extends GenericRepositoryImplTest {
```

Comment:
Why are extending from this test `GenericRepositoryImplTest`? Looks like it is unnecessary